.changeset/sixty-pigs-shave.md

```diff
@@ -0,0 +1,23 @@
+---
+'@backstage/integration': minor
+'@backstage/backend-common': patch
+'@backstage/plugin-catalog-backend-module-github': patch
+'@backstage/plugin-catalog-import': patch
+'@backstage/plugin-git-release-manager': patch
+'@backstage/plugin-github-actions': patch
+'@backstage/plugin-github-issues': patch
+'@backstage/plugin-github-pull-requests-board': patch
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-techdocs-module-addons-contrib': patch
+---
+
+Replaces in-code uses of `GitHub` with `Github` and deprecates old versions.
```

Comment:
Split this up and use the first part of this changeset for all the `patch` packages and make a separate changeset for the integrations package with the `Deprecates:` section :)

Reply:
adjusted